plugins/linux-pipewire/pipewire.c

```diff
@@ -697,7 +724,23 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 		&pod_builder, SPA_TYPE_OBJECT_ParamBuffers, SPA_PARAM_Buffers,
 		SPA_PARAM_BUFFERS_dataType, SPA_POD_Int(buffer_types));
 
+#if PW_CHECK_VERSION(0, 3, 62)
+	if (check_pw_version(&obs_pw->server_version, 0, 3, 62)) {
+		/* Video transformation */
+		params[3] = spa_pod_builder_add_object(
+			&pod_builder, SPA_TYPE_OBJECT_ParamMeta, SPA_PARAM_Meta,
+			SPA_PARAM_META_type,
+			SPA_POD_Id(SPA_META_VideoTransform),
+			SPA_PARAM_META_size,
+			SPA_POD_Int(sizeof(struct spa_meta_videotransform)));
+
+		pw_stream_update_params(obs_pw->stream, params, 4);
```

Comment:
nit, maybe we should use a counter now so we can move `pw_stream_update_params` out of the `ifdef`

Like `params[paramCnt++] = soa_pod_builder_add..` and `pw_stream_update_params(obs_pw->stream, params, paramCnt);`

Reply:
Done